src/time_tagger/hardware_properties/dao.py

```diff
@@ -0,0 +1,35 @@
+from dataclasses import dataclass
+import TimeTagger as TT
+
+
+@dataclass
+class SetTriggerLevelParams:
+    time_tagger_network_proxy: object
+    channels_voltage: dict[int, float]
+
+
+class TimeTaggerHardwarePropertiesDao:
+    def __init__(self) -> None:
```

Comment:
I think we can remove this, as we're just `pass`ing  the execution

Reply:
When I removed it I got an error because of the block indentation. I think Python needs at least something to understand that the scope of the function has ended.